### PR TITLE
Black hole

### PR DESCRIPTION
**Pull request by DJSteins**
Fixes the black hole power up.

### CODE REVIEW

**Comment by DJSteins** [comment]
[Refine blackhole powerup](https://trello.com/c/KhkLYkZw/337-refine-blackhole-powerup)

File: Assets/Script/PowerUp/GravityWellBehaviour.cs [hunk 1]
```diff
@@ -71,23 +71,16 @@ private void OnTriggerEnter(Collider other)
...
-                {
-                    possiblePlayer = carController.PlayerInformation.PlayerNumber;
-                }
-                catch (NullReferenceException ex)
-                {
-                    print("NullReferenceException caught in GravityWellBehevaiour.OnTriggerEnter");
-                    return;
-                }
+                // appears to be the only way to get the car controller out of the collider
+                PlayerController playerController = other.gameObject.transform.parent.parent.gameObject.GetComponent<PlayerController>();
```
Comment (m4rtint) [inline Assets/Script/PowerUp/GravityWellBehaviour.cs]: Might be better to use this
https://docs.unity3d.com/ScriptReference/GameObject.GetComponentInParent.html

Comment (DJSteins) [reply Assets/Script/PowerUp/GravityWellBehaviour.cs]: Tried that and it didn't work :(

Comment (DJSteins) [reply Assets/Script/PowerUp/GravityWellBehaviour.cs]: Actually I tried it with CarController... maybe it works with PlayerController though.

File: Assets/Script/PowerUp/GravityWellBehaviour.cs [hunk 2]
```diff
@@ -71,23 +71,15 @@ private void OnTriggerEnter(Collider other)
...
-                {
-                    possiblePlayer = carController.PlayerInformation.PlayerNumber;
-                }
-                catch (NullReferenceException ex)
-                {
-                    print("NullReferenceException caught in GravityWellBehevaiour.OnTriggerEnter");
-                    return;
-                }
+                PlayerController playerController = other.GetComponentInParent<PlayerController>();
+                CarController carController = playerController.Car;
```
Comment (m4rtint) [inline Assets/Script/PowerUp/GravityWellBehaviour.cs]: Why not just do GetComponentInParent<CarController> ?

Comment (mikeslgoh) [reply Assets/Script/PowerUp/GravityWellBehaviour.cs]: @DJSteins do you have time to change this so that we're able to merge this?

Comment (DJSteins) [reply Assets/Script/PowerUp/GravityWellBehaviour.cs]: I get the following error when I try that:
`NullReferenceException: Object reference not set to an instance of an object
GravityWellBehaviour.OnTriggerEnter (UnityEngine.Collider other) (at Assets/Script/PowerUp/GravityWellBehaviour.cs:75)`

Comment (DJSteins) [reply Assets/Script/PowerUp/GravityWellBehaviour.cs]: Specifically trying `CarController carController = other.GetComponentInParent<CarController>();` does not work

**Pull request merged**